src/app/component-library/catalog/contract-offer-mini-list/contract-offer-mini-list.component.html

```diff
@@ -21,6 +21,14 @@
     </mat-progress-spinner>
   </div>
 
+  <property-grid
+    [columns]="1"
+    [props]="
+      contractOfferIdGroup(contractOffer.contractOfferId)
+    "></property-grid>
+
+  <br />
```

Comment:
Is there really no better way to do this? Like wrapping it in a `div` or something
Is this even necessary?